docs/catalog/web/minify-web-assets

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Can you resubmit the Web Patterns in a different pull request? We assign these out to SMEs and it will be easier to split along the category and help move them through more quickly.

Reply:
Yes @dubrie , will move this